app/assets/templates/comment-stream_tpl.jst.hbs

```diff
@@ -29,7 +29,7 @@
             class="new-comment" id="new-comment-on-{{id}}" method="post">
 
         <textarea class="comment-box form-control mention-textarea"
-                  id="comment_text_on_{{id}}" name="text" rows="1" required placeholder="{{t "stream.comment"}}" />
+                  id="comment_text_on_{{id}}" name="text" rows="1" required placeholder="{{t "stream.comment"}}"> </textarea>
```

Comment:
The space between opening and closing textarea tags maybe a problem as it will add a stray space to the default comment.

Reply:
fixed